### DISCUSSION TITLE
[❄️ Snowflake Official] New roadmap entry - V1 date announcement

### DISCUSSION THREAD

**Discussion by sfc-gh-asawicki** [Snowflake-Labs/terraform-provider-snowflake#3159] [Announcements]
We have updated our publicly available roadmap. It's available [here](https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/main/ROADMAP.md#25102024-project-state-overview).

If there won't be any major obstacles or critical issues we aim to release V1 on **December 9th**.

If you have any questions/doubts, please contact us through GitHub.

**EDIT**: V1 will be released on Wednesday/Thursday this week (**December 11th/12th**). We will add more details with the release. Stay tuned!

**EDIT 2**: V1 was released yesterday. Please check the new roadmap entry here: https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/main/ROADMAP.md#13122024-v1-release-update.

**Comment by jonathanneo** [comment]
Hi @sfc-gh-asawicki just checking in - is the team on-track to releasing V1 on December 9th? 

Where will V1 be released to? Will it be available on Terraform Providers and for me to pull down using the standard way, for example: 

```hcl
terraform {
  required_providers {
    snowflake = {
      source = "Snowflake-Labs/snowflake"
      version = "1.0.0"
    }
  }
}
```

**Reply by sfc-gh-asawicki** [reply]
We are on track for now.

Yes, the standard way will work. :)

**Reply by jonathanneo** [reply]
Thank you! 🙏 

And it's assumed that it's 9th December 2024, and not 2025, right? ;)

**Reply by sfc-gh-asawicki** [reply]
You have found our loophole... ;)

Jokes aside: yes, 2024, so in less than two weeks. :)

**Reply by liamjamesfoley** [reply]
is 0.100 going to be the same as 1.0?

**Reply by sfc-gh-asawicki** [reply]
@liamjamesfoley, no. There will be some changes between that latest 0.x.x (not necessarily 0.100.0!) and v1.0.0. We will communicate them.

**Comment by liamjamesfoley** [comment]
🎉 Wooohooo
https://github.com/Snowflake-Labs/terraform-provider-snowflake/pull/3282

**Reply by jonathanneo** [reply]
Great work @sfc-gh-asawicki and team! I'll take it for a test drive 😉

**Reply by sfc-gh-asawicki** [reply]
We will share more details about the V1 release in the upcoming days and plan to update the roadmap soon. But for now, please give it a go. The team's main priority now is helping with the migration, so please reach out to us with any problems with the migration process or V1 itself.
